isotptun.c

```diff
@@ -118,14 +144,11 @@ int main(int argc, char **argv)
 	unsigned char buffer[BUF_LEN];
 	static char name[IFNAMSIZ] = DEFAULT_NAME;
 	int nbytes;
-
-	signal(SIGTERM, sigterm);
-	signal(SIGHUP, sigterm);
-	signal(SIGINT, sigterm);
+	int run_as_daemon = 1;
 
```

Comment:
This is from slcand. I now looked, and there is little use of `bool`. Even isotptun.c already at least variables `running` and `verbose` defined as `int`, so use of `bool` would be inconsistent.